clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystem.java

```diff
@@ -213,7 +214,7 @@ public FileOutStream getOutStream(long fileId, ClientOptions options) throws IOE
    */
   public boolean mkdirs(TachyonURI path) throws FileAlreadyExistException, InvalidPathException,
       IOException {
-    return mkdirs(path, true);
+    return mkdirs(path, false);
```

Comment:
@jsimsa @calvinjia  Created a new issue 1038 for this as there are many "mkdirs" in the code base. We need to decide whether we refactor all of them or just the convenient function in this file.